### DISCUSSION TITLE
webhook local push not working

### DISCUSSION THREAD

**Discussion by hwansday** [community/community#37196] [Enterprise]
github enterprise webhook setup
Webhook work when push operation is in progress on github enterprise website.
Webhook does not work when commit and push work in local
How should I solve it?
plz~

**Comment by ossanna16** [comment]
Hi there @hwansday and welcome to our community! Thank you for asking a great question 🙂 

While you are waiting for a reply, here are a few things you can do:

* To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
* To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
